src/main/scala/nl.knaw.dans.easy.bag2deposit/DepositPropertiesFactory.scala

```diff
@@ -86,7 +86,7 @@ case class DepositPropertiesFactory(configuration: Configuration, idType: IdType
           addProperty("dataverse.id-identifier", doi.replaceAll(".*/", ""))
           addProperty("dataverse.id-authority", configuration.dataverseIdAutority)
         case URN =>
-          addProperty("dataverse.id-identifier", urn)
+          addProperty("dataverse.id-identifier", bagInfo.baseUrn.getOrElse(urn))
```

Comment:
this does not use the bag-index, it retrieves the baseUrn from the bag-info.txt. This is not present yet, and it won't be for source=VAULT bags